.travis.yml

```diff
@@ -28,3 +28,4 @@ perl:
   - 5.16
 script:
   - dzil smoke --release --author
+  - grunt test-ia
```

Comment:
We need to figure out how to install the dependencies for this. It will probably require some experimentation.

We can probably use node and npm from apt? Then add grunt and bower commands to the install block?

Also need to make sure phantomjs is in the `$PATH`